frontend/src/components/UserProfilePanels/SettingsTab.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {
+  Button, Form, FormGroup, Label, Input, FormText,
+} from 'reactstrap';
+import './SettingsTab.css';
+
+const SettingsTab = () => (
+  <div className="settings-panel">
+    <h2>Password Settings</h2>
+    <Form>
+      <FormGroup>
+        <Label for="oldPassword" hidden>Old Password</Label>
+        <Input type="password" name="password" id="oldPassword" placeholder="Old Password" />
```

Comment:
Might be nice if these did the placeholder magic that the other forms on the site do. But we can always come back to change that down the road, probably by building some custom components for form inputs.